src/Symfony/Bridge/Doctrine/Validator/DoctrineInitializer.php

```diff
@@ -28,7 +28,7 @@ public function __construct(ManagerRegistry $registry)
         $this->registry = $registry;
     }
 
-    public function initialize($object)
+    public function initialize(object $object)
```

Comment:
With this change, DoctrineBridge 5.0 will become incompatible with Validator 4.4. We should either revert this line or update the composer.json file of DoctrineBridge.